.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- Use the first line to close issues. For example:
+    Resolves #93
+-->
+
+<!-- Describe the changes in the pull request. If behaviour has changed,
+describe the previous and new behaviour -->
+
+#### Pre-Merge Checklist:
+- [ ] Code is documented.
+- [ ] Code has automated tests. <!-- If you fixed a bug that was previously
+undetected by tests, it would now be detected. -->
+- [ ] Zero compiler warnings with `-Wall -Wextra`  <!-- Note: enforced by CI check -->
+- [ ] Formatted with `clang-format`                <!-- Note: enforced by CI check -->
```

Comment:
I guess not. The goal is to give information to the submitter without having it appear in the output.